### PR TITLE
updating Hugo image README

### PR DESCRIPTION
**Pull request by SharpRake**
This PR is just to update the README for the Hugo image. This update provides some more context around the application setup, corrects a few broken commands, and removes the unnecessary **Image Variants** section (now in a new tab).

I haven't made a PR here in a while so let me know if I did anything incorrectly.

### Image Size

- [x] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

Notes:

### Image Vulnerabilities

- [x] The Grype vulnerability scan returned 0 CVE(s).
- [ ] The Grype vulnerability scan returned > 0 CVE(s) (please explain in the notes).

Notes:

### Image Tagging

- [ ] The image is _not_ tagged with version tags.
- [x] The image is tagged with :latest
- [ ] The image is not tagged with :latest (please explain in the notes).

Notes:

### Basic Testing - K8s cluster

- [x] The container image was successfully loaded into a kind cluster.
- [ ] The container image could not be loaded into a kind cluster (please explain in the notes).

Notes:

### Basic Testing - Package/Application

- [x] The application is accessible to the user/cluster/etc. after start-up.
- [ ] The application is not accessible to the user/cluster/etc. after start-up. (please explain in the notes).

Notes:

### Helm

- [ ] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.
- [ ] A Helm chart has been provided and the container image is not working with the chart (please explain in the notes).
- [x] A Helm chart was not provided.

Notes:

### Processor Architectures

- [ ] The image was built and tested for x86_64.
- [ ] The image could not be built for x86_64 (please explain in the notes).
- [ ] The image was built and tested for aarch64.
- [x] The image could not be built for aarch64. (please explain in the notes).

Notes:

### Functional Testing + Documentation

- [ ] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes:

### Environment Testing + Documentation

- [x] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.
- [ ] The container image has been tested successfully on a public cloud provider (AWS, GCP, Azure).
- [ ] The container image has not been tested successfully on a public cloud provider (AWS, GCP, Azure) (please explain in the notes).

Notes:

### Version

- [x] The package version is the latest version of the package.  The latest tag points to this version.
- [ ] The package version is the not the latest version of the package (please explain in the notes).

Notes:

### Dev Tag Availability

- [x] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ] There is not a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base') (please explain in the notes).

Notes:

### Access Control + Authentication

- [x] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
- [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [ ] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).

### ENTRYPOINT

- [x] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]
- [ ] applications/servers/utilities not set to call main program with no arguments e.g. [redis-server] (please explain in the notes)
- [ ] base images leave empty.
- [ ] base image and not empty (please explain in the notes).
- [ ] dev variants is set to entrypoint script that falls back to system.
- [ ] dev variants is not set to entrypoint script that falls back to system (please explain in the notes).

### CMD

- [x] For server applications give arguments to start in daemon mode (may be empty)
- [ ] For utilities/tooling bring up help e.g. `–help`
- [ ] For base images with a shell, call it e.g. [/bin/sh]

### Environment Variables

- [x] Environment variables added.
- [ ] Environment variables not added and not required.

### SIGTERM

- [x] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [x] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [x] A README file has been provided and it follows the README template.

### CODE REVIEW

File: images/hugo/README.md [hunk 1]
```diff
@@ -42,25 +35,53 @@ hugo v0.119.0-b84644c008e0dc2c4b67bd69cccf87a41a03937e linux/amd64 BuildDate=202
 
 ## Application Setup for End Users
 
-Here is an example using the Hugo image to run the
-["quickstart"](https://gohugo.io/getting-started/quick-start/#commands) locally:
+The following is an example of using the Hugo image locally. It's based on the official [Hugo "quickstart"](https://gohugo.io/getting-started/quick-start/#commands) example.
```
Comment (ltagliaferri) [inline images/hugo/README.md:38]: It annoys me so much that they call it "quick start" in the documentation but you run it as `quickstart`

File: images/hugo/README.md [hunk 2]
```diff
@@ -42,25 +35,53 @@ hugo v0.119.0-b84644c008e0dc2c4b67bd69cccf87a41a03937e linux/amd64 BuildDate=202
...
+
+To begin, start a shell in the Hugo "dev" container:
```
Comment (ltagliaferri) [inline images/hugo/README.md:40]:
```suggestion
To begin, start a shell in the Hugo developer (dev) container:
```

File: images/hugo/README.md [hunk 3]
```diff
@@ -42,25 +35,53 @@ hugo v0.119.0-b84644c008e0dc2c4b67bd69cccf87a41a03937e linux/amd64 BuildDate=202
...
+```shell
+hugo serve --bind 0.0.0.0 --port 8080
+```
 
-If you're interested in enterprise support, SLAs, and access to older tags, [get in touch](https://www.chainguard.dev/chainguard-images).
+Now open your browser to [localhost:8080](http://localhost:8080) to visit the sample site.
 
+When finished, you can press `CTRL + C` to stop the Hugo server from running, and then `CTRL + D` to exit the container shell.
 
+If you're interested in enterprise support, SLAs, and access to older tags, [get in touch](https://www.chainguard.dev/chainguard-images).
```
Comment (ltagliaferri) [inline images/hugo/README.md:87]: What about the contact us form?

```suggestion
If you're interested in enterprise support, SLAs, and access to older tags, [get in touch](https://www.chainguard.dev/contact?utm_source=docs).
```

**Review by ltagliaferri — COMMENTED** [review]
Thanks, Mark, some minor comments here. Also, in the top section, I think "Hugo" should be title cased in the "hugo process" part

> This is a minimal [Hugo](https://gohugo.io/) image. The image only contains `hugo` and supporting libraries. The hugo process starts in `/hugo` by default so this directory may be initialized with the Hugo site to serve.

**Review by mamccorm — APPROVED** [review]
Nice improvements!

**Pull request merged**